Dockerfile

```diff
@@ -34,10 +33,6 @@ COPY ./src ./src
 # build for release
 RUN mvn clean install -Dmaven.test.skip=true 
 
-# our final base image
-#FROM eclipse-temurin:18.0.1_10-jre
-
-#FROM eclipse-temurin:19_36-jre
 
 FROM eclipse-temurin:latest
```

Comment:
Don't use latest if you don't check for compatibility in a automatic way

Reply:
This has been taken care